sqlglot/dialects/presto.py

```diff
@@ -196,6 +196,17 @@ def _to_int(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
+def _parse_to_char(args: t.List) -> exp.TimeToStr:
```

Comment:
sorry, why are we doing teradata here?

Reply:
Because this is under the "teradata functions" section in both presto's and trino's [docs](https://prestodb.io/docs/current/functions/teradata.html?highlight=to_char#to_char) and the time formats are different from presto's `TIME_MAPPING`, i.e. the format is what Teradata uses.